WBooks/Views/Login/LoginView.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  LoginView.swift
+//  WBooks
+//
+//  Created by Florencia Rosental on 27/02/2018.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import Core
+
+class LoginView: UIView, NibLoadable {
+    
+    @IBOutlet weak var woloxLogoImage: UIImageView!
+    @IBOutlet weak var loginButton: UIButton!
+    @IBOutlet weak var logOutButton: UIButton!
+    @IBOutlet weak var statusText: UILabel!
+    @IBOutlet weak var fondo: UIImageView!
```

Comment:
Are you sure you need all this outlets in the code? I think the `woloxLogoImage` is not needed by code because the image is set in the xib.